### PR TITLE
fix: Display byte axis values using humanReadableBytes [DET-9906]

### PR DESCRIPTION
**Pull request by mapmeld**
## Description

In trial's Profiler tab, charts with Bytes values should be run through our `humanReadableBytes` function, so we see "1.0 GB" instead of "1.00 e+10" Bytes

A separate time axis issue is connected to a backend / query issue we saw in July

## Test Plan

Open an experiment with profiler tab (I use `/det/experiments/1788/profiler`). Scroll down to the System Metrics chart.

Use the dropdown to select each metric. If the axis is "Bytes" or "bytes/s", the human-readable number is displayed:

Screen Shot 2023-10-18 at 12 28 50 PM

Axis labels such as "Gigabytes" or "gigabit/s" are not modified.

## Checklist

- [x] Changes have been manually QA'd
- [x] User-facing API changes need the "User-facing API Change" label.
- [x] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [x] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

File: webui/react/src/pages/TrialDetails/Profiles/Charts/SystemMetricChart.tsx [hunk 1]
```diff
@@ -102,7 +103,11 @@ const SystemMetricChart: React.FC<ChartProps> = ({ trial }) => {
         xAxis={XAxisDomain.Time}
         xLabel="Time"
         yLabel={yLabel}
-        yTickValues={getScientificNotationTickValues}
+        yTickValues={
+          yLabel.substring(0, 5).toLowerCase() === 'bytes'
```
Comment (ashtonG) [inline webui/react/src/pages/TrialDetails/Profiles/Charts/SystemMetricChart.tsx:107]: maybe instead:
```suggestion
          /^bytes/i.test(yLabel)
```
or
```suggestion
          yLabel.toLowerCase().startsWith('bytes')
```

Comment (EmilyBonar) [inline webui/react/src/pages/TrialDetails/Profiles/Charts/SystemMetricChart.tsx:107]: I prefer using regex for situations like this, which would look something like `/^bytes/i.test(yLabel)`, but it's not a big deal

**Review by EmilyBonar — APPROVED** [review]
Looks good!

File: webui/react/src/pages/TrialDetails/Profiles/Charts/SystemMetricChart.tsx (hunk 1 above)
Comment (EmilyBonar) [reply webui/react/src/pages/TrialDetails/Profiles/Charts/SystemMetricChart.tsx:107]: image

Comment (mapmeld) [reply webui/react/src/pages/TrialDetails/Profiles/Charts/SystemMetricChart.tsx]: Thanks ! Glad to join regex collective mind  👋 🎩

**Pull request merged**